{"name": "test_fail_auth", "status": "passed", "description": "Проверка что аутентификация не пройдет при верном пользователе\n        и кривом пароле", "start": 1590592384983, "stop": 1590592387698, "uuid": "8a023bd4-c60f-486b-b933-460f57e564bc", "historyId": "d215a16673445028a97261b01aa5db10", "testCaseId": "ddc8a9b47fcfc11344dc2bd4956f1b7f", "fullName": "tests.test_ui.Test_UI_Auth#test_fail_auth", "labels": [{"name": "story", "value": "main"}, {"name": "tag", "value": "main"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_ui"}, {"name": "subSuite", "value": "Test_UI_Auth"}, {"name": "host", "value": "DESKTOP-4CMVFT5"}, {"name": "thread", "value": "26464-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_ui"}]}